src/app.py

```
@@ -28,6 +29,27 @@
 logging.basicConfig(level=logging.DEBUG if DEBUG_MODE else logging.INFO)
 logger = logging.getLogger(__name__)
 
+# Enable verbose logging of response bodies if VERBOSE env variable is set
+VERBOSE_MODE = os.getenv("VERBOSE", "0").lower() in ("1", "true", "yes")
+
+
+class LogResponseBodyMiddleware(BaseHTTPMiddleware):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)